### PR TITLE
replace license headers

### LINKED ISSUE #231: update license header
Per CNCF requirements, replacing MIT License with Apache

### PR DESCRIPTION
**Pull request by dmitsh**
Closes #231

### CODE REVIEW

**Comment by daixiang0** [comment]
Please update `gen-proto` logic to use new license.

**Comment by dmitsh** [comment]
I actually don't see dapr/proto/ to be used, and I was planning to propose removing this directory.

**Comment by dmitsh** [comment]
> I actually don't see dapr/proto/ to be used, and I was planning to propose removing this directory.

How do we use dapr/proto/ in go-sdk?
Also, there is PROTO_ROOT in the Makefile, which seems not being used.

/cc @yaron2

**Review by mukundansundar — APPROVED** [review]

**Review by daixiang0 — APPROVED** [review]
LGTM.

@yaron2 wait for proto feedback.

**Review by artursouza — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,12 +4,8 @@ Thank you for your interest in Dapr Go SDK!
 
 This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution.
 
-For details, visit https://cla.microsoft.com.
-
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repositories using our CLA.
```
Comment (mukundansundar) [inline CONTRIBUTING.md:7]: @dmitsh Please remove this para also. As CLA bot will not be used right?

**Review by yaron2 — APPROVED** [review]

**Pull request merged**